values/src/main/java/org/spine3/time/LocalDates.java

```diff
@@ -45,4 +46,22 @@ public static LocalDate today() {
                                           .build();
         return result;
     }
+
+    /**
+     * Obtains the date in the future after adding the specified amount of days.
+     */
+    public static LocalDate inDays(int daysDelta) {
+        final Calendar calendar = Calendar.getInstance();
```

Comment:
How about `java.time.LocalDate.now().plusDays(daysDelta)`? And then converting it to our `LocalDate`?